packages/components/src/button/index.stories.jsx

```diff
@@ -1,5 +1,8 @@
 import { action } from '@storybook/addon-actions';
 import Button from '.';
+// Provide theme-default CSS variables from the Sidebar styles
+// eslint-disable-next-line no-restricted-imports
+import 'calypso/my-sites/sidebar/style.scss';
```

Comment:
I don't think we should use Calypso imports in the Components package. How about only adding the necessary variables in this story's SCSS file?

Reply:
I was wondering about this but because its in a story file I didn't give it much attention. it's seems like a circular dependency.